src/components/NewNursery/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { Container, Grid, Typography, useTheme } from '@mui/material';
```

Comment:
reusable component for seedbank and nursery or this ok?
pretty much boilerplate

Reply:
Yes, we could create a common component. Not sure if it will be really useful because it will be used by this two pages only, what do you think?

Reply:
Yes, I agree. This is fine.